shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-postgresql/src/main/java/org/apache/shardingsphere/data/pipeline/postgresql/ingest/PostgreSQLInventoryDumper.java

```diff
@@ -41,4 +43,16 @@ protected PreparedStatement createPreparedStatement(final Connection connection,
         result.setFetchSize(1);
         return result;
     }
+    
+    @Override
+    protected Object readValue(final ResultSet resultSet, final int index) throws SQLException {
+        if (isPgMoneyType(resultSet, index)) {
+            return resultSet.getBigDecimal(index);
+        }
```

Comment:
Return `java.lang.Double` later, and translate to `java.math.BigDecimal` on write into target. Refactor it later.